internal/app/machined/pkg/runtime/v1alpha1/bootloader/grub/grub.go

```diff
@@ -167,6 +167,15 @@ func (g *Grub) Install(fallback string, config interface{}, sequence runtime.Seq
 		}
 	}
 
+	log.Println("writing u-boot-sunxi-with-spl.bin to disk")
+
+	b, err := ioutil.ReadFile("/usr/install/u-boot-sunxi-with-spl.bin")
+
+	_, err = dev.Device().WriteAt(b, 1024*8)
```

Comment:
This doesn't actually seem to be working. I have to manually `dd` this file right now. Not sure why.